client/my-sites/site-settings/style.scss

```diff
@@ -436,11 +436,7 @@
 	margin: 16px 36px 0;
 }
 
-.site-settings__info-link-container {
-	float: right;
```

Comment:
This creates a bunch of oddities all over Settings.

| Before | After |
| --- | --- |
| <img width="503" alt="screen shot 2018-05-24 at 17 32 00" src="https://user-images.githubusercontent.com/2070010/40498979-7a9bf046-5f78-11e8-9ea7-9451940fde16.png"> | <img width="502" alt="screen shot 2018-05-24 at 17 32 08" src="https://user-images.githubusercontent.com/2070010/40498996-8612c85a-5f78-11e8-817a-0b4bc83c863a.png"> |

Reply:
Thank you. I believe the reason is because this PR is just batch one. Batch 2 and 3 will change all of those other icons, which effectively eliminates all use of `.site-settings__info-link-container` once they're all in.

Therefore, we'll need to review and merge all three together and deploy at the same time. Other ideas?